src/test/java/harness/utils/TestUtils.java

```diff
@@ -3,14 +3,16 @@
 import static org.junit.jupiter.api.Assertions.assertEquals;
 
 import bio.terra.cli.serialization.userfacing.UFResource;
-import java.util.Random;
+import java.util.UUID;
 
 public class TestUtils {
-  private static final Random RANDOM = new Random();
-
-  public static String appendRandomNumber(String string) {
-    // This might be used for BQ datasets, so can't have "-". (BQ dataset names can't have "-".)
-    return string + RANDOM.nextInt(10000);
+  // appendRandomNumber: definition copied from Workspace Manager
+  public static String appendRandomNumber(String prefix) {
```

Comment:
Maybe rename this to `appendRandomString` or something similar? This isn't just numbers anymore

Reply:
makes sense, done